bfvmm/src/vmcs/src/vmcs_intel_x64.cpp

```diff
@@ -347,7 +349,7 @@ vmcs_intel_x64::create_vmcs_region()
 vmcs_error::type
 vmcs_intel_x64::release_vmxon_region()
 {
-    // m_memory_manager->free_page(m_vmcs_region);
+    delete m_vmcs_region;
```

Comment:
please change to a make_unique so that we do not have to delete manually. I would like to stick to the "no naked new" rule if possible. In this case, you might be able to use std::array, but if that doesn't work, make_unique<char[]>(4096) will allocate a C style char array of the size your looking for.